### ISSUE TITLE
Required permissions for co-management

### ISSUE THREAD

**Issue by TrentSteenholdt** [MicrosoftDocs/SCCMdocs#626]
What do you do when these steps don't work?

So following these steps, get to the ConfigMgr wizard step and it's grayed out? Running latest SCCM 1802 with latest KB but am stuck. I have a support call open with Micrsosoft and they too are also stumped?! 

I'm looking to just do co-management with domain joined devices so haven't configured the Gateway etc.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 07e9b780-6cad-d3f3-d0f2-e13a577890cc
* Version Independent ID: 18c7f0aa-884a-349b-e416-74d02adc4474
* Content: [Co-management for Windows 10 devices - Configuration Manager](https://docs.microsoft.com/en-us/sccm/core/clients/manage/co-management-overview)
* Content Source: [sccm/core/clients/manage/co-management-overview.md](https://github.com/Microsoft/SCCMdocs/blob/master/sccm/core/clients/manage/co-management-overview.md)
* Product: **configuration-manager**
* GitHub Login: @aczechowski
* Microsoft Alias: **aaroncz**

**Comment by aczechowski** [comment]
Thanks for taking the time to share this with the Configuration Manager documentation team. I’m sorry that you’re having problems, and I want to make sure it gets to the right people that can help. There are a couple of options available to provide the best level of support for this:  

- Post on [Configuration Manager forums](https://social.technet.microsoft.com/Forums/en-US/home?category=ConfigMgrCB). These are moderated by our community MVPs who are very experienced,   knowledgeable, and helpful.  

- Use [Microsoft Support](https://aka.ms/cmcbsupport) to search for the issue, or open a case with professional support.   

- Contact your Microsoft Premier team, such as a Premier Field Engineer or Technical Account Manager.

I'm closing this issue as there's nothing actionable for docs at this time. If you have feedback on the contents of the article, please reopen this issue.

**Comment by TrentSteenholdt** [comment]
I'm sorry @aczechowski but there is clearly a missing piece to the documentation that should be added... It has to do with permissions! 

It turns out the SCCM console in build 1802 (with or without the latest KB) has been changed so the the co-management feature is only available (not greyed out) to an 'Full Administrator' with ALL security scopes. The Default security scope is not enough, but no where in the documentation you have here have you suggested you need to be a 'Full Administrator' on all security scopes. This is contrary to all other Cloud services where the Default scope is enough and 1709 where this wasn't an issue.

I'd recommend that you add at least a note to this documentation to say you must be an administrator to all security scopes. I'm also not the only person with this issue: 
https://configurationmanager.uservoice.com/forums/300492-ideas/suggestions/33766552-unable-configure-co-management-grayed-out-on-1802

I'll make a comment on this to help people get co-management enabled.

Quite disappointing that this was just closed with little to no care. Clearly the instructions to do co-management are not clear enough for people to get this enabled.

**Comment by aczechowski** [comment]
Thank you @TrentSteenholdt for providing further details so that I can understand this is not just a support request, but something specific that we need to include in the documentation. I've clarified the title and reopened the issue. I confirmed with our engineering team that these permissions are required.

Regarding your final comment, I'll also note: the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/), which outlines the expectations for community interactions in and around docs.microsoft.com, is designed to help "provide a welcoming and inspiring community for all." As I stated earlier, I'm sorry that you are having difficulties with this feature, and I want to improve the documentation to help you and others be successful, but this is not a support forum. I'm sorry if you had different expectations.
